jetstream/core/orchestrator.py

```diff
@@ -780,6 +773,10 @@ def _detokenize_thread(self, idx: int):
             # Return some output samples.
             request.enqueue_samples(results)
             if request.complete.all():
+              if self._metrics_collector:
+                self._metrics_collector.get_request_output_length().observe(
+                    result_tokens.get_result_at_slot(slot).lengths
```

Comment:
Since it's streaming response, this length is 1. You need to add a counter to count the # of output tokens till the request completes.

Reply:
result_tokens only has 1 token for each step.